### PR TITLE
Issue #1805

### PR DESCRIPTION
**Pull request by thefex**
Implemented top Projects header UI according to mockups. 
Implemented search functionallity - iOS.

### CODE REVIEW

**Comment by amulware** [comment]
This still needs the following changes:

- Top projects should disappear while showing a filtered list
- While searching, the search bar should be floating at the top, even if the user scrolls through the list

File: Ross/Ross.csproj [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
-    <CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
+    <CodesignEntitlements></CodesignEntitlements>
```
Comment (amulware) [inline Ross/Ross.csproj]: Please remove this change.

File: Ross/Theme/Style.ProjectList.cs [hunk 2]
```diff
@@ -22,35 +22,35 @@ public static void HeaderLabel(UILabel v)
 
             public static void ProjectLabel(UILabel v)
             {
-                v.Font = UIFont.FromName("HelveticaNeue", 17f);
+                v.Font = UIFont.FromName("HelveticaNeue", 15f);
```
Comment (amulware) [inline Ross/Theme/Style.ProjectList.cs]: Please use the `Font` class to obtain the correct font (and same below).

File: Ross/Ross.csproj (hunk 1 above)
Comment (thefex) [inline Ross/Ross.csproj]: This just remove need for provisioning profile on iPhone simulator. The change is done only for "simulator" debug profile.

**Comment by thefex** [comment]
"Top projects should disappear while showing a filtered list" - this is related to other issue, fixed by one of my pull requests for Android. After merging Android PR with fix - it should be quick fix with few lines of iOS code.

File: Ross/Ross.csproj (hunk 1 above)
Comment (amulware) [inline Ross/Ross.csproj]: Fair enough.

**Comment by amulware** [comment]
The referenced pull request was merged, and top projects disappear on android when searching as desired. As @thefex says, making this work in iOS as well should not be hard.

@yaremam Please merge/rebase the changes here locally and implement that behaviour (and other comments) for a new PR. :)

**Pull request closed without merging**